### DISCUSSION TITLE
使用react-dnd, react-dnd-html5-backend 报 probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"'

### DISCUSSION THREAD

**Discussion by chuangwei** [umijs/umi#7850] [Q&A]
使用react-dnd, react-dnd-html5-backend 报 probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"'
请问要怎么配置

**Comment by samsonCao** [comment]
i have the same problem

**Comment by samsonCao** [comment]
ERROR in ./node_modules/@react-dnd/asap/dist/esm/TaskFactory.mjs 2:0-167
Module not found: Error: Can't resolve '/Users/sam/Desktop/iyb-yw/iyb-cube-admin/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/createClass' in '/Users/sam/Desktop/iyb-yw/iyb-cube-admin/node_modules/@react-dnd/asap/dist/esm'
Did you mean 'createClass.js'?
BREAKING CHANGE: The request '/Users/sam/Desktop/iyb-yw/iyb-cube-admin/node_modules/@umijs/babel-preset-umi/node_modules/@babel/runtime/helpers/esm/createClass' failed to resolve only because it was resolved as fully specified
(probably because the origin is a '*.mjs' file or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
 @ ./node_modules/@react-dnd/asap/dist/esm/index.mjs 4:0-34 4:0-34
 @ ./node_modules/dnd-core/dist/esm/HandlerRegistryImpl.js 86:0-39 217:6-10
 @ ./node_modules/dnd-core/dist/esm/DragDropManagerImpl.js 27:0-60 67:54-73
 @ ./node_modules/dnd-core/dist/esm/factories.js 1:0-60 3:20-39
 @ ./node_modules/dnd-core/dist/esm/index.js 2:0-28 2:0-28
 @ ./node_modules/react-dnd/dist/esm/common/DndContext.js 2:0-49 18:21-42
 @ ./node_modules/react-dnd/dist/esm/common/index.js 1:0-29 1:0-29
 @ ./node_modules/react-dnd/dist/esm/index.js 1:0-25 1:0-25
 @ ./src/.umi/.cache/.mfsu/mf-va_react-dnd.js 1:0-26 1:0-26
 @ container entry ./react-dnd[0]

webpack compiled with 6 errors
null
Error: build failed
    at /Users/sam/Desktop/iyb-yw/iyb-cube-admin/node_modules/@umijs/bundler-webpack/lib/index.js:172:29
    at /Users/sam/Desktop/iyb-yw/iyb-cube-admin/node_modules/@umijs/deps/compiled/webpack/5/bundle5.js:114591:14
    at processQueueWorker (/Users/sam/Desktop/iyb-yw/iyb-cube-admin/node_modules/@umijs/deps/compiled/webpack/5/bundle5.js:114529:6)
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
